<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lista_ordini_window</class>
 <widget class="QMainWindow" name="lista_ordini_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista degli ordini</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_titolo">
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lista degli ordini</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="table_ordini">
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cliente</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data e ora</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ammonto</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Spedire a</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prodotti inclusi</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_num_ordini">
        <property name="text">
         <string>Ci sono &lt;num_ordini&gt; ordini nella lista.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="text">
         <string>Chiudi</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
